users/views.py

```diff
@@ -17,6 +17,88 @@
 
 
 class LoginAPIView(APIView):
+    """
+    Endpoint for user login.
+    This endpoint allows users to login by providing their email and password.
+    Parameters:
+    - `request`: HTTP request object.
+    Returns:
+    - 200: Successful login. Returns the refresh and access tokens.
+    - 400: Bad request. Returns a message indicating that the user was not found.
+    - 401: Unauthorized. Returns a message indicating that the password is incorrect.
+    Example request data:
+    {
+        "email": "john@example.com",
+        "password": "strongPassword123!"
+    }
+    Example successful response:
+    HTTP 200 OK
+    {
+        "refresh": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9...",
+        "access": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9..."
+    }
+    Example error response (user not found):
+    HTTP 400 Bad Request
+    {
+        "message": "User not found"
+    }
+    Example error response (wrong password):
+    HTTP 401 Unauthorized
+    {
+        "message": "Wrong password"
+    }
+    """
+
+    @swagger_auto_schema(
+        security=[],
```

Comment:
Maybe we can move everything from @swagger_auto_schema(...) into another file it will improve readability of the code

Reply:
Done :)